dotnet/src/webdriver/Response.cs

```diff
@@ -56,57 +56,69 @@ public Response(SessionId sessionId)
             }
         }
 
-        private Response(Dictionary<string, object> rawResponse)
+        /// <summary>
+        /// Returns a new <see cref="Response"/> from a JSON-encoded string.
+        /// </summary>
+        /// <param name="responseJson">The JSON string to deserialize into a <see cref="Response"/>.</param>
+        /// <returns>A <see cref="Response"/> object described by the JSON string.</returns>
+        public static Response FromJson(string responseJson)
         {
+            Dictionary<string, object> rawResponse = JsonSerializer.Deserialize<Dictionary<string, object>>(responseJson, s_jsonSerializerOptions)
+                ?? throw new WebDriverException("JSON success response returned \"null\" value");
+
+            var @this = new Response();
```

Comment:
Variable name seems ugly, perhaps just `response` is better.

Reply:
I plan to propose some follow-up changes after this PR is merged, and this variable name makes the diffs as clean as possible. I can re-name in a follow-up, or I can change it here if that's preferred.

Reply:
Here and forget, please.

Reply:
renamed to `response`